app/src/main/res/values/strings.xml

```diff
@@ -993,11 +993,11 @@
     <string name="notif_channel_proxy_failure">Proxy Alerts</string>
     <string name="notif_channel_download">Download Alerts</string>
 
-    <string name="notif_channel_desc_vpn_notification">Inform when RethinkDNS is active and running in the background</string>
+    <string name="notif_channel_desc_vpn_notification">Inform when RethinkDNS is up and running in the background</string>
     <string name="notif_channel_desc_vpn_failure">Inform when RethinkDNS is stopped by Android or by another VPN app</string>
-    <string name="notif_channel_desc_firewall_alerts">Alert on Firewall errors and anomalies</string>
-    <string name="notif_channel_desc_proxy_failure">Alert on Proxy errors and anomalies</string>
-    <string name="notif_channed_desc_download">Alert on Download Progress for On-device blocklists</string>
+    <string name="notif_channel_desc_firewall_alerts">Alert for Firewall errors and anomalies</string>
+    <string name="notif_channel_desc_proxy_failure">Alert for Proxy errors and anomalies</string>
+    <string name="notif_channed_desc_download">Alert of Download Progress for On-device blocklists</string>
```

Comment:
`on` is valid, no?

Reply:
That would place the alert onto something.
`Upon` maybe, but that is en_UK centric.

What exactly does this do? I don't really know.